### DISCUSSION TITLE
Random Album dynamic playlist

### DISCUSSION THREAD

**Discussion by jipux** [strawberrymusicplayer/strawberry#1215] [Ideas]
Possible?
preserving the original tracklist order
Adding it to the Smart Playlist section

**Comment by daethlor** [comment]
I started to use Navidrome specifically for this feature.  Would love to see it in Strawberry as I sometimes I'd rather play a random album.

**Comment by jonaski** [comment]
Currently not possible, suggest it on: https://forum.strawberrymusicplayer.org/category/6/feature-suggestions
